android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -69,35 +74,128 @@
 	public static final int PLUGIN_STATE_ON = 1;
 	public static final int PLUGIN_STATE_ON_DEMAND = 2;
 
-	private static enum reloadTypes {
+	private boolean disableTextSelection = false;
```

Comment:
Looks like @hansemannn has just done disableContextMenu in iOS (and not disableTextSelection). May want to limit it to the single property like he did.